app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -126,6 +126,7 @@ def auth_hash_data
         {
           discord: {
             uid: auth_hash[:uid],
+            discriminator: auth_hash[:extra][:raw_info][:discriminator],
```

Comment:
I think you also need to store the `username`. A _Discord tag_ is `username#discriminator`. Might make sense to simply store this as a `discord_tag` column.